invenio_requests/services/schemas.py

```diff
@@ -120,3 +123,47 @@ class RequestEventSchema(BaseRecordSchema, OneOfSchema):
     def get_obj_type(self, obj):
         """Return key of type_schemas to use given object to dump."""
         return obj.type
+
+
+class EventPermissionShema(Schema):
```

Comment:
It looks to me very Comment specific, probably rename to:
```suggestion
class CommentPermissionShema(Schema):
```